.gitignore

```diff
@@ -1,2 +1,9 @@
+# Don't add any custom patterns for platform-specific files like .DS_Store
+# or generated by custom development tools like .vscode or .idea that don't
+# affect other developers. Instead add those custom patterns to your global
+# .gitignore file that can be configured by setting core.excludesFile.
+# See also: https://git-scm.com/docs/gitignore
+
+# Build directories (or a symbolic link, i.e. without a trailing slash)
 /target
-/.direnv
```

Comment:
you shouldn't remove this line as the `.envrc` is still in this repo.

keep this line.

Reply:
I'm sorry, I didn't know that `.direnv` and `.envrc` are connected.